features/generator_specs/generator_specs.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Generator spec
+
+    RSpec spec are normally generated alongside other application components.
+    For instance, `rails generate model` will also generate an RSpec spec file
+    for the model but you can also use your own generator. See
+    [customizing your workflow](https://guides.rubyonrails.org/generators.html#customizing-your-workflow)
```

Comment:
These should be inlined into one line for relish after.

Reply:
https://github.com/rspec/rspec-rails/pull/2217/commits/db35f412140daa7834d74c389f0192fa94d9fbf0